frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/object-service-dashboards.spec.ts

```diff
@@ -0,0 +1,134 @@
+const testName = `test-${Math.random()
+  .toString(36)
+  .replace(/[^a-z]+/g, '')
+  .substr(0, 5)}`;
+
+const testBucket = {
+  apiVersion: 'objectbucket.io/v1alpha1',
+  kind: 'ObjectBucketClaim',
+  metadata: {
+    namespace: 'openshift-storage',
+    name: testName,
+  },
+  spec: {
+    ssl: false,
+    storageClassName: 'openshift-storage.noobaa.io',
+    generateBucketName: 'test-bucket',
+  },
+};
+
+const installStorageCluster = () => {
+  cy.login();
+  cy.visit('/');
+  cy.install();
+};
+
+describe('Tests Buckets Card', () => {
+  before(() => {
+    installStorageCluster();
+  });
+
+  beforeEach(() => {
+    cy.visit('/');
+    cy.byLegacyTestID('horizontal-link-Object Service').click();
+  });
+
+  it('Test at least one noobaa bucket is present', () => {
+    cy.byTestID('1-obc-test')
```

Comment:
This assumes exactly `1 NooBaa Bucket` for the test, only for the time being. Will replace this with a better alternative once I figure out how to get the number of "NooBaa Bucket" via CLI.

Reply:
Noobaa buckets are queried via Prometheus.

Reply:
Fixed in https://github.com/openshift/console/commit/41a827f93d807aaf95c05dba3cf89595f6838fa6.